hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -110,30 +109,34 @@
 	private final Supplier<FCMap<MerkleEntityAssociation, MerkleTokenRelStatus>> tokenAssociations;
 
 	private final PropertySource properties;
-
+	private SpecialFileSystem specialFileSystem;
```

Comment:
Although it won't matter now (since only `File0.0.150` will ever exist in the disk-based file system; and its existence is ensured at startup), this should be a `Supplier` in case we ever decide to create new disk-backed files at runtime. Addressed in #658.